Lib/python/pyabc.i

```diff
@@ -1,14 +1,12 @@
 %define %pythonabc(Type, Abc)
   %feature("python:abc", Abc) Type;
 %enddef
-%pythoncode %{if _swig_python_version_info[0:2] >= (3, 3):
+%pythoncode %{
     import collections.abc
-else:
```

Comment:
Should the `import` be right after the `%{` on the same line, like the existing code is?  I don't remember the details of how indentation inside `%pythoncode` is handled, but keeping it like the existing code should definitely avoid misindentation in the generation Python code.